### ISSUE TITLE
Add pause button to disable running monitors

### ISSUE THREAD

**Issue by hubdotcom** [Runbook/runbook#382]
Otherwise they have to be deleted!

**Comment by madflojo** [comment]
:+1:

This should be do able by sending a delete message to `bridge.py` but not actually deleting it from the database.

**Comment by hubdotcom** [comment]
Yes, perhaps it get's flagged with a date base "deleted" column then a cron fully removes any still flagged after 60 minutes

**Comment by madflojo** [comment]
We can actually use a `status` key to show the Runbook as **Paused**. The main thing with the `bridge.py` process getting a delete message is that will trigger it to remove the monitor from Redis as if it was deleted. But from the UI it would still just show as **Paused**.

Once someone clicked resume or edited the monitor in any way it would then resync and start again. Something to think about is in this scenario it would be best to not resume the monitor after being edited, only after resume.
